src/components/Header.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <header class="fixed top-0 w-100" :class="[lights ? 'z-3' : 'z-max', {'alt-header': routeName === 'series' || routeName === 'media', 'box-shadow-umi': routeName !== 'media'}]">
+  <header class="fixed top-0 w-100" :class="[lights ? 'z-3' : 'z-max', {'alt-header': routeName === 'series' || routeName === 'media', 'box-shadow-umi': routeName !== 'media'},{'darkTheme':darkTheme}]">
```

Comment:
We actually mostly have a dark theme header! 😄 It's the `alt-header` class. We should rename that class to `dark-header`, and change the class to `darkTheme || routeName === 'series' || routeName === 'media'`. There will probably need a couple changes to make it work perfectly, but that one is battle tested for styles!